docs/operator-ci-yaml.md

```diff
@@ -0,0 +1,58 @@
+# Operator Publishing / Review settings
+
+Each operator might have `ci.yaml` configuration file to be present in an operator directory (for example `community-operators/aqua/ci.yaml`). This configuration file is used by [community-operators](https://github.com/operator-framework/community-operators) pipeline to setup various features like `reviewers` or `operator versioning`.
+
+> **Note:**
+    One can create or modify `ci.yaml` file with a new operator version. This operation can be done in the same PR with other operator changes. 
+
+## Reviewers
+
+If you want to accelerate publishing your changes, consider adding yourself and others you trust to the `reviewers` list. If the author of PR will be in that list, changes she/he made will be taken as authorized changes. This will be the indicator for our pipeline that the PR is ready to merge automatically. 
+
+> **Note:**
+    If an author of PR is not in `reviewers` list or not in `ci.yaml` on `main` branch, PR will not be merged automatically. For security reasons, we are not checking reviewers from a fork.
```

Comment:
Can you please elaborate more on the sentence `For security reasons, we are not checking reviewers from a fork.`? What is meant by that?

Reply:
It was pasted from the old details and was supposed to be removed from this documentation. I removed it now.